gnocchi/rest/__init__.py

```diff
@@ -145,7 +145,10 @@ def PositiveNotNullInt(value):
 
 
 def Timespan(value):
```

Comment:
Yes in order to return a proper error from voluptuous and not have https://github.com/gnocchixyz/gnocchi/issues/195